.github/workflows/ci.yml

```diff
@@ -170,7 +173,7 @@ jobs:
       - name: Set up Helm
         uses: azure/setup-helm@v1
         with:
-          version: v3.6.0
+          version: v3.11.2
```

Comment:
`azure/setup-helm` should be moved to `v3` instead of `v1`.

Reply:
Done.